common/scylla_image_setup

```diff
@@ -17,14 +17,13 @@
 import os
 import sys
 import pathlib
-sys.path.append('/opt/scylladb/scripts')
-from scylla_util import is_redhat_variant
+import distro
 from lib.scylla_cloud import get_cloud_instance, is_gce
 from subprocess import run
 
 if __name__ == '__main__':
     # On Ubuntu, we configure CPU scaling while AMI building time
-    if is_redhat_variant():
+    if distro.id() == 'centos':
```

Comment:
this is not as not redhat.... (i.e. now we support images only for centos and ubuntu, but down the road we might have more, i.e. rocky based image ?)